tox.ini

```diff
@@ -1,30 +1,28 @@
 [tox]
 downloadcache = {toxworkdir}/cache/
 envlist = # sort by django version, next by python version
-    {core,example,docs}-py{35,36,37,38}-django22,
-    {core,example,docs}-py{36,37,38}-django30,
-    {core,example,docs}-py{36,37,38,39}-django31,
-    {core,example,docs}-py{36,37,38,39,310}-django32,
-    {core,example,docs}-py{38,39,310}-django40,
-    {core,example,docs}-py{38,39,310}-djangomain,
+    {core,example,docs}-py{38,39,310}-django32,
+    {core,example,docs}-py{38,39,310,311}-django41,
+    {core,example,docs}-py{38,39,310,311}-django42,
+    {core,example,docs}-py{310,311,312}-django50,
+    {core,example,docs}-py{310,311,312}-djangomain,
```

Comment:
My 0.02 but unless we intent on regularly testing against django's devel Im not sure if I would keep `{core,example,docs}-py{310,311,312}-djangomain`.  Not now - just a thought going forward.  I realize this is out of scope for this MR

Reply:
Leaving it as is.